src/molecule/test/unit/test_platforms.py

```diff
@@ -28,10 +28,23 @@ def _instance(config_instance):
     return platforms.Platforms(config_instance)
 
 
+@pytest.fixture
+def _instance_with_platform_name(config_instance):
+    return platforms.Platforms(config_instance, platform_name="instance-1")
+
+
 def test_instances_property(_instance):
     x = [
         {"groups": ["foo", "bar"], "name": "instance-1", "children": ["child1"]},
         {"groups": ["baz", "foo"], "name": "instance-2", "children": ["child2"]},
     ]
 
     assert x == _instance.instances
+
+
+def test_instances_property_with_platform_name(_instance_with_platform_name):
+    x = [
+        {"groups": ["foo", "bar"], "name": "instance-1", "children": ["child1"]},
```

Comment:
`instance-1` is hardcoded in the test body and the detached fixture. It is best to keep such coupled things in the same place. I think that having an extra fixture is not warranted in this case. But if it was, this should've definitely gone to "parameterize" where it could be evident/visible that these values are connected.

Reply:
Thank you for pointing it out.

Two parametrizing test functions are added.

`test_instances_property_with_platform_name_instance_1` and `test_instances_property_with_platform_name_instance_2` is to avoid conditional assertions.

Let me know if you have any concerns.